lib/rom/repository/command_compiler.rb

```diff
@@ -208,7 +208,10 @@ def finalize_command_class(klass, relation)
 
         klass.extend_for_relation(relation) if klass.restrictable
 
-        plugins.each { |plugin| klass.use(plugin) }
+        plugins.each do |(plugin, block)|
+          klass.use(plugin) 
+          klass.instance_eval(&block) if block
```

Comment:
ooh, block configuration of plugins at registration?  I've been considering attempting to add that :)

Although, that we generally register the plugin ourselves means it's not typically useful for an end user, outside of registering some non-standard configs (ie,  `register :short_pages, Pagination,  -> { page_count 5 }` or something like) ... which makes it seem obscure.  :/